3.2/Dockerfile

```diff
@@ -33,26 +30,25 @@ RUN yum install -y centos-release-scl && \
     chmod g+w -R /var/opt/rh/rh-mongodb32/lib/mongodb && \
     chmod -R g+rwx /var/lib/mongodb
 
-# Get prefix path and path to scripts rather than hard-code them in scripts
-ENV CONTAINER_SCRIPTS_PATH=/usr/share/container-scripts/mongodb \
+ENV MONGODB_VERSION=3.2 \
+    HOME=/var/lib/mongodb \
+    # Set paths to avoid hard-coding them in scripts.
+    CONTAINER_SCRIPTS_PATH=/usr/share/container-scripts/mongodb \
```

Comment:
we generally put ENV front and center in the dockerfiles because it contains critical information people go looking for when trying to understand the env defined for an image.  I view it like comments.  I think they're better suited to be lumped w/ the labels.

and i disagree that these are more likely to change than the packages you're installing.  the env variables are almost never going to change.  the packages will, next time someone asks us to add another common mongo utility.  (or we need to chmod something else, or add some other step to the RUN block)